modal-js/src/sandbox.ts

```diff
@@ -156,6 +210,35 @@ export class Sandbox {
       }
     }
   }
+
+  async tunnels(timeout = 50000): Promise<Record<number, Tunnel>> {
```

Comment:
Should this have a docstring? `timeout` might be a bit surprising as an option, since if it is increased to above 1 minute, the request will fail with the global server gRPC timeout

Reply:
Yeah I added it to align with [the Python client](https://modal.com/docs/reference/modal.Sandbox#tunnels), added docstring now too!

Reply:
Interesting, I think that might be an oversight on the Python side then. Not sure we want timeout to be a user-selectable parameter when it behaves a bit surprisingly. But that's fine for now.